src/Microsoft.WSMan.Management/ConfigProvider.cs

```diff
@@ -1977,8 +1977,8 @@ private void NewItemCreateComputerConnection(string Name)
                 helper.CreateWsManConnection(parametersetName, dynParams.ConnectionURI, dynParams.Port, Name, dynParams.ApplicationName, dynParams.UseSSL, dynParams.Authentication, dynParams.SessionOption, this.Credential, dynParams.CertificateThumbprint);
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-linux`
Invoke-WebRequest tests.Denial of service.Charset Parsing
```powershell
Expected the actual value to be greater than 5, but got 0.272884181596744.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 1961
1961:             $pathologicalRatio | Should -BeGreaterThan 5
```